content/en/_index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Findy Agency
+---
+
+{{< blocks/cover title="Findy Agency" image_anchor="top" height="full" >}}
+<a class="btn btn-lg btn-primary me-3 mb-4" href="/blog/">
+  Blog <i class="fas fa-arrow-alt-circle-right ms-2"></i>
+</a>
+<a class="btn btn-lg btn-secondary me-3 mb-4" href="https://github.com/findy-network">
+  Archive <i class="fab fa-github ms-2 "></i>
+</a>
+<p class="lead mt-5">Findy Agency is an open-source project for a decentralized identity agency.
+OP Lab developed it from 2019 to 2024. The project is no longer actively maintained,
```

Comment:
Just a thought, we can discuss online about this:

OP Lab actively developed and maintained the project from 2019 to 2024. The project is no longer maintained, but codes (CODE-LINK) are still available. The project will be pivoted and split where FIDO2 & Passkey-related parts will get a new home, the legacy part of the SSI will be modernized, and it will similarly get a new home and name. **The work is currently in progress.**

The team will update this page and report ongoing progress on **BLOG-PAGE-LINK**. The blogs will also include cryptography posts, digital identities, and related topics about software development in general.

Reply:
The project will be pivoted and the work will continue with new goals and mission. Follow the blog for updates.